### ISSUE TITLE
key never change

### ISSUE THREAD

**Issue by richardjuan** [nfc-tools/mfcuk#32]
hello,
i use "mfcuk -C -R 0 -v 3 -o dump" to try to find a key with the last version of mfcuk
it seams not to work cause the key is always 000000000 in the verbose output

-----------------------------------------------------
Let me entertain you!
    uid: d3e215f3
   type: 08
    key: 000000000000
  block: 03
diff Nt: 3351
  auths: 9997
-----------------------------------------------------
is it normal? or do i have to change something in the source code to make it change the key?
thanks in advance.

**Comment by ilfuriano** [comment]
I have the same issue.

**Comment by richardjuan** [comment]
i just waited and everything went ok ;)

**Comment by ilfuriano** [comment]
Which version of mfoc and libnfc did you use?

**Comment by richardjuan** [comment]
not sure, but i get the one in this tutorial or above :/

http://www.george-smart.co.uk/wiki/NFC_Experiments

**Comment by theforcedk** [comment]
Same issue. OS X, libnfc 1.7.1, mfcuk 0.3.8

**Comment by shoaly** [comment]
@richardjuan i have the same issue, are you confirm mfcuk is running ok even the key never change?

**Comment by richardjuan** [comment]
@shoaly  yep, everything is ok. btw some reader implementation ( garrage door, buildings entry ) look like they don't care about what's in the key and only look for the uid :)

**Comment by SamDecrock** [comment]
@richardjuan How long did it eventually take?

**Comment by SamDecrock** [comment]
After 2 days I got this error:

```
Let me entertain you!
    uid: 985d588e
   type: 08
    key: 000000000000
  block: 03
diff Nt: 65536
  auths: 128671
-----------------------------------------------------
mfcuk: ERROR: mfcuk_key_recovery_block() (error code=0x09)
```

**Comment by ChunJyeBehBeh** [comment]
@SamDecrock Hello, at the end have you successfully crack the card?

**Comment by SamDecrock** [comment]
@ChunJyeBehBeh I did using the mfoc hardnested fork. I made a [write up](https://medium.com/@decrocksam/cracking-mifare-classic-nfc-cards-using-the-hardnested-attack-506aab3ea305) about it.
